src/ocm/components/clusterConfiguration/staticIp/data/formDataToInfraEnvField.ts

```diff
@@ -183,5 +183,7 @@ export const networkWideToInfraEnvField = (
   }
   const currentFormData = formDataFromInfraEnvField(staticNetworkConfig);
   fixHostIps(networkWide, currentFormData.hosts);
+  //Transform vlanId to number
+  networkWide.vlanId !== '' ? (networkWide.vlanId = Number(networkWide.vlanId)) : '';
```

Comment:
```suggestion
  networkWide.vlanId ??= Number(networkWide.vlanId);
```

Reply:
BTW, `networkWide.vlanId !== '' /*could be true when nw.vlanId is undefined*/`
and then `Number(undefined) === NaN`

Reply:
@jkilzi not works well.
 Only if the value of x is nullish then the value of y will be assigned to x that means if the value of x is null or undefined then the value of y will be assigned to x. But in this case we have a value in networkWide.vlanId and therefore the null condition is not true.
On the other hand, it does not pass the test and gives an error: "Parsing error: expression expected".

Reply:
ack, I think we had a mid-air collision. I too just figured out what I proposed wasn't good.
What about this? `networkWide.vlanId = typeof networkWide.vlanId === 'undefined' ? 0 : Number(networkWide.vlanId);`